src/System.Runtime.CompilerServices.Unsafe/ref/Configurations.props

```diff
@@ -0,0 +1,9 @@
+﻿<?xml version="1.0" encoding="utf-8"?>
+<Project ToolsVersion="14.0" DefaultTargets="Build" xmlns="http://schemas.microsoft.com/developer/msbuild/2003">
+  <PropertyGroup>
+    <BuildConfigurations>
+      netstandard1.0;
+      netstandard;
```

Comment:
Yes, we do catch dependency cycles.  The [VerifyClosure](https://github.com/dotnet/buildtools/blob/master/src/Microsoft.DotNet.Build.Tasks.Packaging/src/VerifyClosure.cs) task runs against [MS.NETCore.App](https://github.com/dotnet/corefx/blob/29939b7fdbdec35a844e2a6c9063f8e4587822a7/pkg/Microsoft.Private.CoreFx.NETCoreApp/Microsoft.Private.CoreFx.NETCoreApp.pkgproj#L84-L104) and catches exactly the type of cycle you suggest.

If we do run into a situation where a low-level assembly needs to use this we can cross-compile for NETCoreApp and have it reference either System.Runtime, or even System.Private.Corelib.  If you'd prefer to undo the factoring I did to allow this project to specify the core-assembly I can do that too.  I left it in because it seemed like the "right thing to do" to build a netstandard version of this library, and because it uncovered a bug in ilasm.

Currently this assembly isn't even in the shared framework FYI.